WeslToml.md

```diff
@@ -0,0 +1,87 @@
+# `wesl.toml` file format
+
+WESL shader projects have a standardized `wesl.toml` file, which describes the project setup. It is used by language servers and linkers[^1].
+
+The format looks as follows. As much of it is optional as possible.
+
+```toml
+[package] # Following the toml convention of grouping properties.
+
+# Optional, used for specifying the name within wesl shaders.
```

Comment:
Change this phrasing to explicitly mention wesl import statements.

Reply:
Name field will not be a thing in favour of automatic name translations and users using explicit names